src/test/java/com/attica/athens/domain/agora/AgoraAuthApiIntegrationTest.java

```diff
@@ -309,4 +320,298 @@ void givenAlreadyEndVote_whenEndAgoraVote_thenReturnsError() throws Exception {
+                    .andExpectAll(
+                            jsonPath("$.success").value(false),
+                            jsonPath("$.response").value(nullValue()),
+                            jsonPath("$.error.code").value(expectedCode),
+                            jsonPath("$.error.message").value(expectedMessage)
+                    );
+        }
+
+        @Test
+        @DisplayName("categoryId가 존재하지 않은경우 예외를 발생시킨다.")
+        void nonExistentCategoryIdThrowsException() throws Exception {
+            // given
+            AgoraCreateRequest request = new AgoraCreateRequest("test-title", 5, 180, "red", 3L);
+            String expectedCode = "1301";
+            String expectedMessage = "Not found category. categoryId: " + request.categoryId();
+
+            // when & then
+            mockMvc.perform(post("/{prefix}/agoras", API_V1_AUTH)
+                            .contentType(MediaType.APPLICATION_JSON)
+                            .content(objectMapper.writeValueAsString(request)))
+                    .andExpect(status().isNotFound())
+                    .andExpectAll(
+                            jsonPath("$.success").value(false),
+                            jsonPath("$.response").value(nullValue()),
+                            jsonPath("$.error.code").value(expectedCode),
+                            jsonPath("$.error.message").value(expectedMessage)
+                    );
+        }
+
+        @Test
+        @DisplayName("아고라를 생성한다.")
+        void createAgora() throws Exception {
+            // given
+            AgoraCreateRequest request = new AgoraCreateRequest("test-agora", 5, 60, "red", 1L);
+
+            // when & then
+            mockMvc.perform(post("/{prefix}/agoras", API_V1_AUTH)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(objectMapper.writeValueAsString(request)))
+                    .andExpect(status().isOk())
+                    .andExpectAll(
+                            jsonPath("$.success").value(true),
+                            jsonPath("$.response.id").value(1),
+                            jsonPath("$.error").value(nullValue())
+                    );
+        }
+    }
+
+    @Nested
+    @Sql(scripts = {
+            "/sql/get-category.sql",
+            "/sql/get-agora.sql",
+            "/sql/get-base-member.sql"
+    })
+    @DisplayName("아고라 참가 테스트")
+    @WithMockCustomUser("EnvironmentalActivist")
+    class participateAgoraTest {
+
+        @BeforeEach
+        void setup() {
+            objectMapper = new ObjectMapper();
+        }
+
+        @Test
+        @DisplayName("찬성 역할로 아고라에 참가한다.")
+        void participateAgoraAsProsRole() throws Exception {
+            // given
+            Long userId = 1L;
+            Long agoraId = 1L;
+            AgoraParticipateRequest request = new AgoraParticipateRequest(
+                    "test-nickname",
+                    1,
+                    AgoraMemberType.PROS);
```

Comment:
p3: 중요하진 않지만 우선 적어봤어요,,
```suggestion
                    AgoraMemberType.PROS
            );
```